Create writer-levels.md (#885)
mattermost/mattermost-handbook: operations/research-and-development/product/technical-writing-team-handbook/writer-levels.md

# Technial Writer Levels

These levels serve to help Technical Writers assess their opportunities for career growth in the Techncial Writing function. The levels below follow a growth track in the art and craft of Technical Writing and not in a person management track.

### Level 1: Technical Writer I

| Leadership Principle | Competencies | Level Expectations |
|---|---|---|

> | Ownership | Scope: "What do I work on?" <br> <br>*Scope is the domain over which you are expected to show ownership.* | Executes to deliver high quality documentation and in-product text updates. |

Reviewer: ```suggestion
| Ownership | Scope: "What do I work on?" <br> <br>*Scope is the domain over which you are expected to demonstrate ownership.* | Executes to deliver high quality documentation and in-product text updates. |
```

Reviewer: If you agree with this update, let's ensure all instances of this sentence are updated for consistency.

Author: This came from the template that HR provided so I am hesitant to change it here for the consistency sake of other departments.

| High-Impact | Supervision: "How much do I rely on my manager?"<br> <br>Community: "How do others work with me?"<br> <br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives detailed instruction on most tasks. Asks appropriate questions, and knows when to stop and ask for help. <br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
| Earn Trust | Influence: "Who do I work with?"<br> <br>*As more people and parts of the organization earn your trust, your influence grows.* | Works effectively in teams to align priorities and manage execution. |
| Customer Obsession | Customer Impact: "How do I bring value?"<br> <br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Considers the customer's perspective when making decisions. Knows when to ask clarifying questions. |

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Regularly participates in team discussions for ongoing feature work.  May lead discussions on smaller topics.<br><br>Learns new skills and establishes goals and context quickly by asking precise questions. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to iterate towards the best possible outcome.* | Regularly participates in team discussions for ongoing feature work. May lead discussions on smaller topics.<br><br>Learns new skills and establishes goals and context quickly by asking precise questions. |
```

Reviewer: If you agree with this update, let's ensure all instances of this sentence are updated for consistency.

Author: This is another one that came from HR.  I will revert the commit.

| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives general instruction on day-to-day work, and detailed instruction on new assignments or unfamiliar tasks.<br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Works effectively in teams to align priorities and manage execution. |
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Understands and applies basic level best practices for customer impact within their daily work. Will ask for input on more complicated topics. |

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Regularly participates in team discussions for ongoing feature work.  May lead discussions on smaller topics.<br><br>Actively seeks out tasks and projects that work towards opportunities for growth. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Regularly participates in team discussions for ongoing feature work. May lead discussions on smaller topics.<br><br>Actively seeks out tasks and projects that work towards opportunities for growth. |
```

### Level 3: Technical Writer II
| Leadership Principle | Competencies | Level Expectations |
|---|---|---|
| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.* | Designs, leads and delivers high impact information architectures, in-product text, and documentation improvements across their area of ownership.	 |
| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives general instruction on new assignments or unfamiliar tasks.<br><br>Participates in cross-functional groups towards building significant new functionality. |
| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Work directly with Product, UX, Engineering, GTM, Operations, and Community to proactivly seek input and deliver on commitments.	 |

> | Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Can navigate moderately complex decisions and thought process about how features and implementations bring customer value, and can make decisions in these areas independently.  Starts to understand and contribute to cross-feature and/or cross-team implications of customer impact.	|

Reviewer: ```suggestion
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Can navigate moderately complex decisions and thought process about how features and implementations bring customer value, and can make decisions in these areas independently. Starts to understand and contribute to cross-feature and/or cross-team implications of customer impact.	|
```

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Drops fluidly into team projects, ramps quickly and leads initiatives to successful outcomes. Leads discussions on small and mid-size topics.  Participates actively in discussion and efforts that cross teams (PM, GMT, Operations, Engineering, UX). <br><br> Actively seeks out tasks and projects that work towards opportunities for growth. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Drops fluidly into team projects, ramps quickly and leads initiatives to successful outcomes. Leads discussions on small and mid-size topics. Participates actively in discussion and efforts that cross teams (PM, GMT, Operations, Engineering, UX). <br><br> Actively seeks out tasks and projects that work towards opportunities for growth. |
```

### Level 4: Technical Writer II

| Leadership Principle | Competencies | Level Expectations |
|---|---|---|
| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.* | Designs, leads and delivers high impact information architectures, in-product text, and documentation improvements across their area of ownership.	 |
| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives minimal instruction.<br><br>Participates in cross-functional groups towards building significant new functionality. |
| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Work directly with Product, UX, Engineering, GTM, Operations, and Community to proactivly seek input and deliver on commitments. |

> | Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Can navigate moderately complex decisions and thought process about how features and implementations bring customer value, and can make decisions in these areas independently.  Starts to understand and contribute to cross-feature and/or cross-team implications of customer impact.	 |

Reviewer: ```suggestion
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Can navigate moderately complex decisions and thought process about how features and implementations bring customer value, and can make decisions in these areas independently. Starts to understand and contribute to cross-feature and/or cross-team implications of customer impact.	 |
```

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Drops fluidly into team projects, ramps quickly and leads features to successful outcomes. Leads discussions on small and mid-size topics.  Participates actively in discussion and efforts that cross teams (PM, GMT, Operations, Engineering, UX). <br><br>Proactively identifies opportunities for growth, intentionally taking on unfamiliar tasks. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Drops fluidly into team projects, ramps quickly and leads features to successful outcomes. Leads discussions on small and mid-size topics. Participates actively in discussion and efforts that cross teams (PM, GMT, Operations, Engineering, UX). <br><br>Proactively identifies opportunities for growth, intentionally taking on unfamiliar tasks. |
```

### Level 5: Senior Technical Writer
| Leadership Principle | Competencies | Level Expectations |
|---|---|---|

> | Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Sets and delivers vision for documentation architecture and high impact features to improve technical writing experiences for across all product lines.	 |

Reviewer 2: ```suggestion
| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Sets and delivers vision for documentation architecture and high impact features to improve technical writing experiences across all product lines.	 |
```

Reviewer 2: This may work or there may be a word missing, it's unclear.

| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Defines new feature assignments for themselves, usually without requiring help. Organizes cross-functional initiatives in building significant new functionality.|
| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.*| Recognized by colleagues as a technical writing authority, passively influencing discussions and behavior, and working in sync with Product, UX, Engineering, Operations, and GTM. |

> | Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Can independently make decisions affecting customer (internal and external) value/impact for complex topics within a product deliverable for the company.  Leads cross-product, cross-feature, and cross-team discussions related to customer value/impact, and can bring the stakeholders to a decision point. |

Reviewer: ```suggestion
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Can independently make decisions affecting customer (internal and external) value/impact for complex topics within a product deliverable for the company. Leads cross-product, cross-feature, and cross-team discussions related to customer value/impact, and can bring the stakeholders to a decision point. |
```

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.*  | Frequently called upon to comment on business discussions. Is very comfortable in a variety of techincal and business discussions, aligns efforts, and develops superior solutions through discussion and analysis. Participates deeply in cross-team efforts. Begins to lead discussions on topics that reach outside of product area of ownership. <br><br>Independently researches new solutions and paradigms to improve team efficiency. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.*  | Frequently called upon to comment on business discussions. Is very comfortable in a variety of techincal and business discussions, aligns efforts, and develops superior solutions through discussion and analysis. Participates deeply in cross-team efforts. Begins to lead discussions on topics that reach outside of product area of ownership. <br><br>Independently researches new solutions and paradigms to improve team efficiency. |
```

| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Highly respected by colleagues and community as a technical authority, actively influencing discussions and behavior with input and suggestions, and working in sync with with Product, UX, Engineering, Operations, and GTM. |
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Can independently make decisions affecting customer (internal and external) value/impact for complex topics within a product deliverable for the company. Leads cross-product, cross-feature, and cross-team discussions related to customer value/impact, and can bring the stakeholders to a decision point. |

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Frequently called upon to comment on business discussions. Is very comfortable in a variety of techincal and business discussions, aligns efforts, and develops superior solutions through discussion and analysis. Participates deeply in cross-team efforts. Begins to lead discussions on topics that reach outside of product area of ownership. <br><br>Independently researches new strategies and paradigms to improve the entire Tech Writing organization and product area of ownership. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Frequently called upon to comment on business discussions. Is very comfortable in a variety of techincal and business discussions, aligns efforts, and develops superior solutions through discussion and analysis. Participates deeply in cross-team efforts. Begins to lead discussions on topics that reach outside of product area of ownership. <br><br>Independently researches new strategies and paradigms to improve the entire Tech Writing organization and product area of ownership. |
```

| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.*| Defines new task assignments for members of other teams, usually without requiring help. Mentors and trains new team members while leading the coordination and management of high impact initiatives.|
| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Frequently sought out by leadership for opinion on documentation and technical writing decisions (both product and technical directions). |
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Has ownership for cross-product customer impact topics. Can anticipate complex issues early in the planning and development processes. Starts to anticipate and resolve cross functional topics and issues. |

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Primary technical writing leader for projects, providing leadership in selecting and guiding these efforts. Works regularly with stakeholders outside R&D. Is often the intial resource to drive efforts for new initiatives. <br><br> Mentors people on technical writing strategy across all of product and engineering. Is a hands on leader for executing on mentorship and training. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Primary technical writing leader for projects, providing leadership in selecting and guiding these efforts. Works regularly with stakeholders outside R&D. Is often the initial resource to drive efforts for new initiatives. <br><br> Mentors people on technical writing strategy across all of product and engineering. Is a hands on leader for executing on mentorship and training. |
```

### Level 8: Principal Technical Writer

| Leadership Principle | Competencies | Level Expectations |
|---|---|---|
| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Owns and defines long-term technical writing practice vision and strategy. |

> | High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Independently defines new initiatives across teams working on the same product or system.  Influences, shapes and can redirect customer and community technical and business discussions, rapidly understanding disparate viewpoints and leading discussions that align thinking and efforts to influence the direction of large scale projects. |

Reviewer: ```suggestion
| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Independently defines new initiatives across teams working on the same product or system. Influences, shapes, and can redirect customer and community technical and business discussions, rapidly understanding disparate viewpoints and leading discussions that align thinking and efforts to influence the direction of large scale projects. |
```

| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Seen as the expert on product area of ownership and technical writing practice. Engages with peers in customer and partner organizations to shape initiatives and plans. |

> | Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Works consistently with R&D, GTM, G&A, and Sales leadership to set goals and deliverables for customer and business value. Represents Technical Writers and other functions in discussions where ICs are not present.  Engages heavily with customers and community to share our product philosopy and strategy. Applies our philosophy to bring value to customer from work done all across the company.|

Reviewer: ```suggestion
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Works consistently with R&D, GTM, G&A, and Sales leadership to set goals and deliverables for customer and business value. Represents Technical Writers and other functions in discussions where ICs are not present. Engages heavily with customers and community to share our product philosophy and strategy. Applies our philosophy to bring value to customer from work done all across the company.|
```

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Works consistently with Mattermost leadership to set technical writing & organizational objectives and direction.  Translates these objectives to clear and actionable projects for execution. <br><br>Develops and oversees all aspects of mentoring and training efforts. Can easily identify individual needs, and find solutions to help people grow and succeed. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Works consistently with Mattermost leadership to set technical writing & organizational objectives and direction. Translates these objectives to clear and actionable projects for execution. <br><br>Develops and oversees all aspects of mentoring and training efforts. Can easily identify individual needs, and find solutions to help people grow and succeed. |
```

### Level 9: Distinguished Technical Writer

| Leadership Principle | Competencies | Level Expectations |
|---|---|---|
| Ownership | Scope: "What do I work on?" <br><br>*Scope is the domain over which you are expected to show ownership.*  | Innovates new strategies and opportunities to vastly improve technical writing practice at Mattermost. Translates learnings from the broader tech world to bring value to Mattermost's customers and community. |

> | High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Conceives, designs, explains, and oversees product direction for nearly any feature or product with no outside direction. Independently executes on cross-team (within R&D) and cross-function (within Mattermost) effort with little or no supervision.  Considered an expert in many domains. Works across a broad set of technical and non-technical efforts.  |

Reviewer: ```suggestion
| High-Impact | Supervision: "How much do I rely on my manager?"<br><br>Community: "How do others work with me?"<br><br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Conceives, designs, explains, and oversees product direction for nearly any feature or product with no outside direction. Independently executes on cross-team (within R&D) and cross-function (within Mattermost) effort with little or no supervision. Considered an expert in many domains. Works across a broad set of technical and non-technical efforts.  |
```

| Earn Trust | Influence: "Who do I work with?"<br><br>*As more people and parts of the organization earn your trust, your influence grows.* | Relied upon as a key resource for stakeholders both inside and outside of Product Management. Engages with people in the broader tech/PM community (outside of Mattermost) to identify and implement best practices within Mattermost.|

> | Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Works consistently with R&D, GTM, G&A, and Sales leadership to set goals and deliverables for customer and business value. Represents Technical Writers and other functions in discussions where ICs are not present.  Engages heavily with customers and community to share our product philosopy and strategy. Applies our philosophy to bring value to customer from work done all across the company.|

Reviewer: ```suggestion
| Customer Obsession | Customer Impact: "How do I bring value?"<br><br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.*  | Works consistently with R&D, GTM, G&A, and Sales leadership to set goals and deliverables for customer and business value. Represents Technical Writers and other functions in discussions where ICs are not present. Engages heavily with customers and community to share our product philosophy and strategy. Applies our philosophy to bring value to customer from work done all across the company.|
```

> | Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Works consistently with Mattermost leadership to set product & organizational objectives and direction.  Translates these objectives to clear and actionable projects for execution. <br><br>Develops and oversees all aspects of mentoring and training efforts. Can easily identify individual needs, and find solutions to help people grow and succeed. |

Reviewer: ```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Works consistently with Mattermost leadership to set product & organizational objectives and direction. Translates these objectives to clear and actionable projects for execution. <br><br>Develops and oversees all aspects of mentoring and training efforts. Can easily identify individual needs, and find solutions to help people grow and succeed. |
```
